src/hardware/reelmagic/player.h

```diff
@@ -0,0 +1,35 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2020-2024  The DOSBox Staging Team
+ *  Copyright (C) 2002-2021  The DOSBox Team
```

Comment:
Same here (zero references to reel magic in DOSBox SVN, and the vogons member who authored the reel magic emulation isn't part of the 'The DOSBox Team' when I checked the SVN `AUTHORS` file or here: https://www.dosbox.com/crew.php).

Reply:
Yeah, ReelMagic has nothing to do with original DOSBox. We should credit the author, though. His name is in the release notes and the first commit of this file.

Reply:
Changed copyright Jon Dennis, same as what's in `player.cpp`